the-experts-BE/app.js

```diff
@@ -33,20 +30,6 @@ if (process.env.NODE_ENV === "production") {
   });
 }
 
-// catch 404 and forward to error handler
-app.use(function(req, res, next) {
-  next(createError(404));
-});
-
-// error handler
-app.use(function(err, req, res, next) {
-  // set locals, only providing error in development
-  res.locals.message = err.message;
-  res.locals.error = req.app.get("env") === "development" ? err : {};
-
-  // render the error page
-  res.status(err.status || 500);
-  res.render("error");
-});
```

Comment:
You still want this function, as it's your 500 error handler. Remember 404 is an error where the route doesn't exist which you don't need to worry about now. 500 is if there is an error somewhere in your server code.

You allso don't want to use `res.render()` as that's for templates. Use `res.send()` or `res.json()` and think about what message you to send back to the front end if there's a 500 error.